### DISCUSSION TITLE
Spring Boot 3.x

### DISCUSSION THREAD

**Discussion by richeyh** [LogNet/grpc-spring-boot-starter#308] [Q&A] [answered]
With the new M4 release of spring-boot found [here](https://github.com/spring-projects/spring-boot/releases/tag/v3.0.0-M4), I was wondering what the upgrade plan for this library would be. I do not see a beta branch or milestone release in this project to use the milestone from spring boot. What is the plan to upgrade this?

**Comment by jvmlet** [comment]
@richeyh, Spring boot 3.0 will be supported,may be  not at the same date as it's released, but definitely yes.

**Reply by richeyh** [reply]
any kind of idea of a timeline of when? also has anyone attempted using the milestone version?

**Comment by jvmlet** [comment, marked as answer]
`5.0.0` has been released.
